op-acceptance-tests/tests/interop/loadtest/initiator.go

```diff
@@ -0,0 +1,80 @@
+package loadtest
+
+import (
+	"math/rand"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/contracts/constants"
+	"github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/interop"
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/txintent"
+	"github.com/ethereum-optimism/optimism/op-service/txplan"
+	"github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var rng = rand.New(rand.NewSource(1234))
+
+type Initiator interface {
+	Initiate(t devtest.T) []types.Message
+}
+
+type ManyMsgsInitiator struct {
+	el          *dsl.L2ELNode
+	eoa         *dsl.EOA
+	eventLogger common.Address
+	counter     *counter
+}
+
+func NewManyMsgsInitiator(funder *dsl.Funder, el *dsl.L2ELNode, eventLogger common.Address) *ManyMsgsInitiator {
+	return &ManyMsgsInitiator{
+		eoa:         funder.NewFundedEOA(eth.MillionEther),
+		el:          el,
+		eventLogger: eventLogger,
+		counter:     new(counter),
+	}
+}
+
+func (in *ManyMsgsInitiator) Initiate(t devtest.T) []types.Message {
+	const numMsgs = 275 // About the max number of msgs we can create before hitting tx size limits.
+	initCalls := make([]txintent.Call, 0, numMsgs)
+	for range numMsgs {
+		initCalls = append(initCalls, interop.RandomInitTrigger(rng, in.eventLogger, rng.Intn(5), rng.Intn(10)))
+	}
+	return buildAndSendInitTx(t, in.eoa, in.el, &txintent.MultiTrigger{
+		Emitter: constants.MultiCall3,
+		Calls:   initCalls,
+	}, txplan.WithStaticNonce(in.counter.Next()))
+}
+
+type LargeMsgInitiator struct {
+	eoa         *dsl.EOA
+	el          *dsl.L2ELNode
+	eventLogger common.Address
+	counter     *counter
+}
+
+func NewLargeMsgInitiator(funder *dsl.Funder, el *dsl.L2ELNode, eventLogger common.Address) *LargeMsgInitiator {
+	return &LargeMsgInitiator{
+		eoa:         funder.NewFundedEOA(eth.MillionEther),
+		el:          el,
+		eventLogger: eventLogger,
+		counter:     new(counter),
+	}
+}
+
+func (lin *LargeMsgInitiator) Initiate(t devtest.T) []types.Message {
+	// TODO: can we create an even larger event without the event logger?
```

Comment:
TODO comments need a ticket attached or the CI job that monitors this will fail

Reply:
TIL, thanks https://github.com/ethereum-optimism/optimism/issues/16039#issue-3078381609